internal/adapters/consul/testdata/multiple-services.json

```diff
@@ -40,6 +40,13 @@
               "x-add-too": "2"
             }
           }
+        },
+        {
+          "Type": 2,
+          "URLRewrite": {
+            "Type": 1,
```

Comment:
The other change I suggested _should_ result in this test breaking since type `1` no longer exists due to the fact that there is now only one `URLRewriteType` constant, `ReplacePrefixMatchURLRewriteType = iota`

Reply:
![Screen Shot 2022-07-13 at 2 30 19 PM](https://user-images.githubusercontent.com/100602640/178816283-06da6686-f0ec-48b2-bcf7-98eb4fbff497.png)

Right on the money, fixing now